# Generated for NaoQi method ALFaceTracker.setWholeBodyOn
# if true, the tracking will be through a Whole Body Process.

# The whole Body state
bool pWholeBodyOn
---


